pay-api/src/pay_api/services/statement.py

```diff
@@ -418,10 +420,7 @@ def _build_statement_summary_for_methods(
         if Statement.is_payment_method_statement(statement_dao, statement_purchases, PaymentMethod.EFT.value):
             summary.update(Statement._populate_statement_summary(statement_dao, statement_purchases, PaymentMethod.EFT))
         if Statement.is_payment_method_statement(statement_dao, statement_purchases, PaymentMethod.PAD.value):
```

Comment:
Looks like PAD and EFT logic is now the same, we can probably reduce duplication here by reviewing and updating the is_payment_method_statement

Reply:
k gonna check what can be simplified on EFT statement update ticket